tecs/tecs.cpp

```diff
@@ -316,7 +315,7 @@ void TECS::_update_throttle_setpoint(const float throttle_cruise, const matrix::
 	// Calculate the throttle demand
 	if (_underspeed_detected) {
 		// always use full throttle to recover from an underspeed condition
```

Comment:
if this change is accepted, you might want to update the comment to read something like "//always use max throttle to recover from an underspeed condition"

Reply:
changed this